src/components/ApplicationsAndServices/homeLayout.spec.ts

```
@@ -0,0 +1,360 @@
+  it('reads the modern categories array', () => {
+    expect(folderCategoryFromDoc({ categories: ['banking'] })).toBe('banking')
+  })
+
+  it('falls back to the legacy singular category', () => {
+    expect(folderCategoryFromDoc({ category: 'cozy' })).toBe('cozy')
+  })
+
+  it('prefers a non-empty categories array over the singular field', () => {
+    expect(folderCategoryFromDoc({ categories: ['health'], category: 'cozy' })).toBe(
+      'health'
+    )
```

ESLint

[error] Replace `folderCategoryFromDoc({·categories:·['health'],·category:·'cozy'·})).toBe(⏎······'health'⏎····` with `⏎······folderCategoryFromDoc({·categories:·['health'],·category:·'cozy'·})⏎····).toBe('health'`

(prettier/prettier)